horovod/run/run.py

```diff
@@ -410,6 +434,9 @@ def parse_args():
     parser.add_argument('-p', '--ssh-port', action='store', dest='ssh_port',
                         type=int, help='SSH port on all the hosts.')
 
+    parser.add_argument('--ssh-ports', action='store', dest='ssh_ports',
```

Comment:
Maybe assert that both `--ssh-ports` and `-ssh-port` aren't set at the same time?